WebRoot/messages/ZaMsg.properties

```diff
@@ -2378,6 +2378,12 @@ BUSY_GET_RETENTION_POLICIES = Getting retention polices ...
 BUSY_CREATE_RETENTION_POLICIES = Creating retention polices ...
 BUSY_DELETE_RETENTION_POLICIES = Deleting retention polices ...
 BUSY_MODIFY_RETENTION_POLICIES = Modifying retention polices ...
+BUSY_GETTING_SYNC_DEVICES = Getting synced devices ...
```

Comment:
What about translations for other languages?
Create another ticket to track it if you are not going to cover the same in this ticket.

Reply:
Added translations